android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.getcapacitor.community.intercom.intercom">
-    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Sorry, I only noticed this removal. Are these permissions not needed anymore?

Reply:
Interestingly I've had no issues with the plugin without this, but I'll add this back as we had this before. Thanks for picking this up